### DISCUSSION TITLE
Currents in Hybrid Code

### DISCUSSION THREAD

**Discussion by Beforerr** [BLAST-WarpX/warpx#4925] [Q&A] [answered]
Could someone help clarify whether the current output is ion current, or is it derived from Ampere-Maxwell's Law?

Thanks 🙇‍♀

**Comment by roelof-groenewald** [comment, marked as answer]
The current output by the diagnostic is just ion current. The electron current can also be output by adding `J_displacement` to the diagnostic data list (note that standard displacement current in Maxwell's equations is calculated the same way as the electron current in the hybrid scheme where we assume the actual displacement current is zero - I hope that makes sense, see [here](https://warpx.readthedocs.io/en/latest/theory/kinetic_fluid_hybrid_model.html#electron-current)).

**Reply by Beforerr** [reply]
Thank you for the explanation. Yes, it make sense.
